todo_app/app.py

```diff
@@ -1,15 +1,81 @@
-from flask import Flask
 
+from flask import Flask, render_template, request, redirect, url_for
+from todo_app.data.session_items import get_item, get_items, add_item, clear_items, clear_item, update_item_status, sort_items
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
 @app.route('/')
-def index():
-    return 'Hello World!'
+@app.route('/<op_message>')
```

Comment:
This solution to including a message from before the redirect works, but you could use Flask's `flash` instead.
https://flask.palletsprojects.com/en/1.1.x/patterns/flashing/

It works using a cookie instead of a path parameter.

The advantage of using a cookie instead means the URL doesn't end up super long from long messages, and also you won't have issues with the user reloading the page and seeing the message again.